arch/arm/soc/st_stm32/stm32f4/Kconfig.defconfig.stm32f446xe

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST STM32F446XE MCU configuration options
+#
+# Copyright (c) 2018 Philémon Jaermann.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F446XE
+
+config SOC
+	string
+	default stm32f446xx
+
+config NUM_IRQS
+	int
+	default 96
+
```

Comment:
I would add here:
```
if GPIO_STM32

config GPIO_STM32_PORTE
	default n

config GPIO_STM32_PORTH
	default n

endif # GPIO_STM32
```

Reply:
I made this change, and pushed an update